src/Tickets/Seating/Tables/Sessions.php

```diff
@@ -120,6 +120,7 @@ protected function get_definition() {
 				`object_id` bigint(20) NOT NULL,
 				`expiration` int(11) NOT NULL,
 				`reservations` longblob DEFAULT '',
+				`expiration_lock` int(1) DEFAULT 0,
```

Comment:
Maybe tinyint or boolean is a better option for the column type ?

Reply:
Good idea, thank you.